### PR TITLE
[IMP] Website: Updating seo documentation

### PR DESCRIPTION
**Pull request by Donapi**
[IMP] Website: Updating seo documentation

taskid-[3499195](https://www.odoo.com/web#id=3499195&cids=1&model=project.task&view_type=form)

### CODE REVIEW

**Review by zulu-odoo — CHANGES_REQUESTED** [review]
Hi Dona,
Please see my review comments. As always, feel free to ignore anything that you disagree with, and I'm available whenever if you want to discuss in person :)
Thank you!

File: content/applications/websites/website/pages/seo.rst [hunk 1]
```diff
@@ -1,640 +1,166 @@
...
-, pt\_PT (Portuguese from Portugal).
+Meta tags are HTML elements that provide information about a webpage to search engines and website
+visitors. They play a crucial role in SEO by helping search engines understand the content and
+context of a webpage and attract visitors with appealing content.
 
-Language Annotation
--------------------
+:guilabel:`Title` tags specify a webpage's title and are displayed as a clickable link in search
+engine results. It should be concise, descriptive, and relevant to the page's content. You can
+update the title tag of your webpage or keep it empty to use the default value.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:30]: updated

File: content/applications/websites/website/pages/seo.rst [hunk 2]
```diff
@@ -1,640 +1,166 @@
...
 
-To let search engines know that the second URL is the French translation of the
-first URL, Odoo will add an HTML link element in the header. In the HTML
-<head> section of the main version, Odoo automatically adds a link
-element pointing to the translated versions of that webpage;
+:guilabel:`Description` tags summarize the web page's content, often displayed in search engine
+results below the title. It is essential for encouraging click-through. You can update the
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:33]: updated and simplified

File: content/applications/websites/website/pages/seo.rst [hunk 3]
```diff
@@ -1,640 +1,166 @@
...
+description tag of your webpage or keep it empty to use the default value based on the page’s
+content.
 
--  <link rel="alternate" hreflang="fr"
-   href="https://www.mywebsite.com\/fr\_FR/shop/product/mon-produit-1"/>
+.. note::
+   The :guilabel:`Preview` card overviews how the title and description tags should appear in search
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:38]: How they "should", as per JQU ...

File: content/applications/websites/website/pages/seo.rst [hunk 4]
```diff
@@ -1,640 +1,166 @@
...
+   results. It also includes the URL of your page.
 
-With this approach:
+Keywords
+--------
 
-- Search engines will redirect to the right language according to the
-  visitor language.
+SEO keywords are specific words or phrases from your website that visitors can search for in search
+engines to find what they're looking for. Creating a list of relevant keywords is essential in any
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:45]: I have been asked by the PO to delete this paragraph

File: content/applications/websites/website/pages/seo.rst [hunk 5]
```diff
@@ -1,640 +1,166 @@
...
+SEO strategy to boost traffic.
 
-- You do not get penalized by search engines if your page is not translated
-  yet. Indeed, it's not a duplicated content, but a different
-  version of the same content.
+Odoo ``<keyword>`` finder helps you see how your keywords are used at the different levels of your
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:46]: It's the context and not the field. I think we could debate about it :)

File: content/applications/websites/website/pages/seo.rst [hunk 6]
```diff
@@ -1,640 +1,166 @@
...
-
-.. note::
-  If the CSS file is very big, Odoo will split it into two smaller
-  files to avoid the 4095 selectors limit per sheet of Internet Explorer.
-  But most themes fit below this limit.
-
-**Minified:** After being pre-processed and concatenated, the resulting
-CSS is minified to reduce its size.
+Odoo automatically compresses images you download to reduce their sizes (lossless compression for
+.PNG and .GIF and lossy compression for .JPG). All images used in Odoo official themes have been
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst]: I discussed it with cwa and updated the sentence

File: content/applications/websites/website/pages/seo.rst [hunk 7]
```diff
@@ -1,640 +1,166 @@
...
+   :alt: automated image compression
 
-Then, a cached version is stored server-side (so we do not have
-to pre-process, concatenate, minify at every request) and browser-side
-(so the same visitor will load the CSS only once for all pages they
-visit).
+.. important::
+   Alt tags are used to describe what’s on your image. Adding keywords as alt tags is essential
+   from an SEO perspective. This description is added to the HTML code of your image, and it is
+   shown when the image cannot be displayed.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:93]: the image right above shows the description field to add the alt tag, that's why it's here

File: content/applications/websites/website/pages/seo.rst [hunk 8]
```diff
@@ -1,640 +1,166 @@
...
-  own server)
+Structured data markup is used to generate rich snippets in search engine results. It is a way for
+website owners to send structured data to search engine robots, helping them understand your content
+and create well-presented search results.
 
-- Split the resource loading on different services allowing to load
-  more resources in parallel (since the Chrome limit of 6 parallel
-  requests is by domain)
+Google supports many rich snippets for content types, including: Reviews, People, Products,
+Businesses, Events and Organizations.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst]: By default. Clarified in the sentence

File: content/applications/websites/website/pages/seo.rst [hunk 9]
```diff
@@ -1,640 +1,166 @@
...
-  for your employees using Odoo)
+Scalability
+-----------
 
-Most visitors of your website will only need the first two bundles,
-resulting in a maximum of two Javascript files to load to render one
-page. As these files are shared across all pages, further clicks by the
-same visitor will not load any other Javascript resource.
+We are cautious about implementing only scalable features. Your website remains fast even if you
+have dozens of thousands of products.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:99]: makes sense!

**Review by zulu-odoo — APPROVED** [review]
Hi Dona,
Looks good to me :)
Thank you!

File: content/applications/websites/website/pages/seo.rst [hunk 10]
```diff
@@ -1,640 +1,173 @@
...
-   concatenated, nor minified. Thus, it's much slower. But it allows you to easily debug with the
-   Chrome debugger as CSS and Javascript resources are not transformed from their original versions.
+Responsive design
+=================
 
-CDN
----
+All Odoo themes rely on the CSS Framework `Bootstrap <https://getbootstrap.com/>`_ to render
+efficiently according to the device: desktop, tablet, or mobile, which positively impacts ranking in
+search engines.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:103]: Good idea!

File: content/applications/websites/website/pages/seo.rst [hunk 11]
```diff
@@ -1,640 +1,173 @@
...
+products, forum posts, and contact addresses. This allows your product pages to be displayed in
+Google using extra information like the price and rating of a product:
 
-.. image:: seo/seo11.png
-  :align: center
+.. image:: seo/data-markup.png
+   :alt: snippets in search engine results
 
-HTML Pages
+robots.txt
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages/seo.rst:125]: We speak about it in the tip in the sitemap section.

**Review by auva-odoo — APPROVED** [review]
@robodoo r+

**Comment by auva-odoo** [comment]
@robodoo r-

**Comment by Donapi** [comment]
@robodoo r+

**Comment by auva-odoo** [comment]
@robodoo r+

**Pull request closed without merging**